test/parallel/test-http2-client-connection-tunnelling.js

```diff
@@ -63,6 +63,12 @@ netServer.listen(0, common.mustCall(() => {
       netServer.close();
     }));
 
+    tlsSocket.once('error', common.expectsError({
+      name: 'Error',
+      code: 'ERR_HTTP2_STREAM_ERROR',
+      message: 'Stream closed with error code NGHTTP2_CANCEL'
+    }));
```

Comment:
I don't think it should error this. I'll see tomorrow.

Reply:
Yeah it's `CONNECT` it shouldn't error `NGHTTP2_CANCEL` or any other nghttp2 error.